### DISCUSSION TITLE
Local font is not showing up in build (nuxt3)

### DISCUSSION THREAD

**Discussion by njiels** [nuxt/nuxt#21017] [Questions]
**`Nuxt3.4.3`**

I've added local fonts which are visible in dev mode, but once I check the build they do not render. When checking the dist folder I saw that the font assets did not come through. What did I do wrong / forget?

My _global.css:
```
@font-face {
    font-family: BebasNeuePro, 'sans-serif';
    font-weight: 400;
    src: url('~assets/fonts/BebasNeuePro-Bold.ttf') format('woff');
}

@font-face {
    font-family: BebasNeueProExp, 'sans-serif';
    font-weight: 400;
    src: url('~assets/fonts/BebasNeuePro-Book.ttf') format('woff');
}

body{
    font-family: 'BebasNeuePro';
}
```

This is working, in build I can see that the body computes the right font-family.

My nuxt.config.ts
```
export default defineNuxtConfig({
    modules: ['@nuxtjs/tailwindcss'],
    css: ['@/assets/_global.css']
  })
```
I am using a very bare nuxt project, the only dependency I've installed is tailwind. Am i forgetting something in the nuxt.config?

https://github.com/njiels/red-het-ov

^ repo of my project

**Comment by maou-shonen** [comment]
This mabey not be the answer,
But can you try `~/assets` instead of `~assets`?

**Reply by njiels** [reply]
This did not work, but it turned out it was a wrongly set tailwind config. My bad!

**Comment by njiels** [comment]
I've fixed it, in tailwind.config.js I had written down the wrong font-family. I changed it to the right one so it looks like this:

```
module.exports = {
  content: [],
  theme: {
    fontFamily: {
      primary: ['BebasNeuePro'],
      secondary: ['BebasNeueProExp']
    },
    extend: {},
  },
  plugins: [],
}
```

**Comment by NoahLawu** [comment]
I had same issue like this but i already following your example code but the local font still not working, here's my code

Can you help me to figure it out the solution of this issue ? Thank you

My main.css file

```
@tailwind base;
@tailwind components;
@tailwind utilities;

@font-face {
  font-family: Bagina Regular,'sans-serif;
  font-style: normal;
  font-weight: normal;
  font-display: swap;
  src: url('/FrontEnd/public/fonts/bagina.woff') format("truetype");
  }
```

My tailwind.config file
```
/** @type {import('tailwindcss').Config} */
export default {
  content: [],
  theme: {
    fontFamily: {
      bagina: ['Bagina Regular'],
    },
  extend:{},
  container: {
      center: true,
      padding: {
        DEFAULT: '1rem',
        sm: '2rem',
        lg: '4rem',
        xl: '5rem',
        '2xl': '6rem',
      },
    },
  },
  plugins: [],
}
```

**Reply by azzumaraydid7** [reply]
any update ?

**Reply by azzumaraydid7** [reply]
there is some fonts is working and some not working

**Reply by leonardotessaroalves** [reply]
wrong font format. Replace TrueType by woff

**Reply by decoin1987** [reply]
I think this is your problem https://tailwindcss.com/docs/font-family

Tailwind does not automatically escape font names for you. If you’re using a font that contains an invalid identifier, wrap it in quotes or escape the invalid characters.

// Add quotes:
  'sans': ['"Exo 2"', ...],

  // ...or escape the space:
  'sans': ['Exo\\ 2', ...],

**Comment by ahadihadi** [comment]
have same issue

**Reply by ahadihadi** [reply]
in my case the problem was name of font files, including ( ) character's, removing parentheses from font file name solved it!

seems silly!
